lib/rules/no-this-before-super.js

```diff
@@ -36,6 +36,8 @@ function isConstructorFunction(node) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
Weird things happen if trying to use `this` or `super` before invoking `super()`, right? Should this be a "problem"?